assembl/views/api2/discussion.py

```diff
@@ -553,13 +560,13 @@ def extract_taxonomy_csv(request):
         else:
             qualify_by_action = "no qualify by action"
         owner_of_the_message = db.query(m.User).filter(m.User.id == query.creator_id).first().name
-        published_on = str(query.creation_date)
+        published_on = str(query.creation_date)[:19]
         harvester = db.query(m.User).filter(m.User.id == extract.owner_id).first().name
-        harvested_on = str(extract.creation_date)
+        harvested_on = str(extract.creation_date)[:19]
         nugget = "Yes" if extract.important else "No"
         extract_info = {
             "Thematic": thematic.encode('utf-8'),
-            "Message": sanitize_html(message).encode('utf-8'),
+            "Message": sanitize_text(message).encode('utf-8'),
```

Comment:
why do you have to change this ?

Reply:
@cedricmessiant I have to change this because when I export the csv, it exports the text with HTML tags inside.